monai/bundle/scripts.py

```diff
@@ -1539,3 +1539,126 @@ def init_bundle(
+        version = bundle.get(meta="version")
+        print(version)
+
+    Args:
+        bundle_name: the name of the bundle. Defaults to None.
+        config_path: file path of the bundle. If a list of file paths is provided,
+            their contents will be merged. Defaults to None.
+        bundle_dir: the target directory to store downloaded bundle.
+            Defaults to the 'bundle' subfolder under `torch.hub.get_dir()`.
+        target directory to store the downloaded data.
+            Default is `bundle` subfolder under `torch.hub.get_dir()`.
+        configs: The name of the config file(s), supporting multiple names.
+            Defaults to "train".
+        kwargs: Additional arguments for download or workflow class instantiation.
+    """
+
+    def __init__(
+        self,
+        bundle_name: str | None = None,
+        config_path: str | Sequence[str] | None = None,
+        bundle_dir: PathLike | None = None,
+        configs: str | Sequence[str] = "train",
+        **kwargs: Any,
+    ) -> None:
+        if bundle_name is None and config_path is None:
+            raise ValueError("Must specify bundle_name or config_path.")
+        configs = ensure_tuple(configs)
+        if "train" in configs:
+            workflow = "train"
+        else:
+            workflow = "infer"
+
+        if config_path is not None:
+            _config_path = Path(ensure_tuple(config_path)[0])
+            if _config_path.is_file():
+                config_file = config_path
+                config_root_path = _config_path.parent
+            else:
+                raise FileNotFoundError(f"Cannot find the config file: {config_path}.")
+        else:
+            download_args = {
+                "version": kwargs.pop("version", None),
+                "args_file": kwargs.pop("args_file", None),
+                "source": kwargs.pop("source", download_source),
+            }
+            download(bundle_name, bundle_dir=bundle_dir, **download_args)
+            bundle_dir = _process_bundle_dir(bundle_dir)
+            config_root_path = bundle_dir / bundle_name / "configs"  # type: ignore
+            if len(configs) > 0:
+                config_file = [
+                    str(config_root_path / _find_bundle_file(config_root_path, _config)) for _config in configs  # type: ignore
+                ]
+            else:
+                config_file = str(config_root_path / _find_bundle_file(config_root_path, configs[0]))  # type: ignore
+
+        logging_file = config_root_path / "logging.conf"
+        self.meta_file = config_root_path / "metadata.json"
+
+        self.workflow = ConfigWorkflow(
+            config_file=config_file,
+            meta_file=str(self.meta_file),
+            logging_file=str(logging_file),
+            workflow=workflow,
+            **kwargs,
+        )
+        self.workflow.initialize()
+
+    def get(self, property: str | None = None, meta: str | None = None) -> Any:
```

Comment:
would be great to support a default value, such as https://github.com/Project-MONAI/MONAI/blob/a86c0e01555727bd848275c7eabc5b9dc26da73d/monai/bundle/config_parser.py#L191

Reply:
Supported now, thanks.